examples/stable_diffusion_v2/docs/en/safety_check.md

```diff
@@ -0,0 +1,39 @@
+### Safety Checker
+
+To validate the safeness of images generated by a model, we implement two safety checkers in `safety_checker_1.py` and `safety_checker_2.py`.
+
+`safety_checker_1.py` applies CLIP to compute the similarity between the given images and a pre-defined list of NSFW concepts (two possible `.yaml` files to choose from). This is consistent with the one used in CompVis's stable diffusion 1.x from `diffusers` implementation. The output is a list of similarity scores, each corresponding to a concept.
+
+`safety_checker_2.py` trains a NSFW classifier with a supervisied approach, taking image features from CLIP as its input. This is consistent with the one used in StabilityAI's stable diffusion 2.x. The output is a number in 0-1, representing the probability of the generated image being NSFW.
+
+You can use the checkers in your image generation pipelines. It is by default included in `text_to_image.py` for checking the safety of stable diffusion outputs. In addition, you can test the checkers on saved images. In `examples/stable_diffusion` directory, please run,
+
+- MindSpore backend
+```
+python tools/safety_checker/safety_checker_1.py --ckpt_path <path-to-model> --image_path_or_dir <path-to-image>
+python tools/safety_checker/safety_checker_2.py --ckpt_path <path-to-model> --image_path_or_dir <path-to-image>
+```
+- PyTorch backend
+```
+python tools/safety_checker/safety_checker_1.py --backend pt --image_path_or_dir <path-to-image>
```

Comment:
Can we combine safety_checker_1.py and safety_checker_1.py into one script like safety_checker.py? And use an argument --version to select which one we would like to use

Reply:
done